### PR TITLE
Fixes koppor/jabref#521 - Allow to drag&drop selected bib entries to other library tabs

### PR DESCRIPTION
**Pull request by GNAQ**
Fixed [https://github.com/koppor/jabref/issues/521](https://github.com/koppor/jabref/issues/521)

by adding a new feature that allows to drag selected entries, and drop them to other opened inactive bib library tabs.

### Behavior Description:

Now when dragging selected entries in main table, JabRef allows user to drop it on another opened library tab, and this item will be copied and added to the dropped library accordingly. Drop to current active tab is disallowed. If there're potential duplicated entries, JabRef will ask user to confirm and choose copy behavior (similar to ctrl-c/v copy and paste).

### Screenshots for UI changes:

[](https://gifyu.com/image/SKPDF)

### Checklist / TODOs :

- [x] Code implement of this new feature
- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: CHANGELOG.md
Comment (Siedlerchr) [inline CHANGELOG.md:19]:
```suggestion
- We added a feature to drag selected entries and drop them to other opened inactive library tabs [koppor521](https://github.com/koppor/jabref/issues/521).
```

**Review by Siedlerchr — APPROVED** [review]
thanks for this PR! Codewise looks already good to me.
Before merging, a second maintainer needs to review your code as well

**Comment by GNAQ** [comment]
> thanks for this PR! Codewise looks already good to me. Before merging, a second maintainer needs to review your code as well

Thank you for your kindly review!

**Comment by koppor** [comment]
Thank you for the PR! We have a small nitpick that `t` should be named `tab` (to have a bit more readable local variable name).

We could not update it for ourselves, because editing by maintainers is disabled:

grafik

We merge nevertheless and will rename later (at some time ^^).

Thank you for the PR!

**Pull request merged**
